ormlitegenerator/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
#? ORMlite isn't _really_ that light. It appears to require an insane amount of configuration.....